### PR TITLE
fix(DB/spell_dbc): Execute Warrior

### PR DESCRIPTION
**Pull request by Stifler82**
## Changes Proposed:
-  Fixed [Execute](https://wowgaming.altervista.org/aowow/?spells=7.1.256&filter=na=execute;cl=1) Spell in All Ranks, Now the spell will not fail.

## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/235
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4961

## Tests Performed:

- Tested in game and working
- Tested on Windows 10 (64)
- **Video Test:** [Execute](https://youtu.be/AOE5w4_6YZw)

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1617549311457174700.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    CHANGE `Description_Lang_zhCN` `Description_Lang_zhCN` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_enTW` `Description_Lang_enTW` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_zhTW` `Description_Lang_zhTW` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_esES` `Description_Lang_esES` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_esMX` `Description_Lang_esMX` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_ruRU` `Description_Lang_ruRU` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_ptPT` `Description_Lang_ptPT` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_ptBR` `Description_Lang_ptBR` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_itIT` `Description_Lang_itIT` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_Unk` `Description_Lang_Unk` text(0) DEFAULT NULL;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1617549311457174700.sql:19]: text(0)?
so no limits?

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617549311457174700.sql:19]: yes, there are description too long in game for spells

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1617549311457174700.sql:19]: I am not sure that the spell_dbc description fields affect the description that you see in the client, but that's ok

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617549311457174700.sql:19]: for example, this warrior description for execute was like this:
`Attempt to finish off a wounded foe, causing ${$m1+$AP*0.2} damage and converting each extra point of rage into $*10;F1 additional damage (up to a maximum cost of 30 rage).  Only usable on enemies tha`
instead
like this:
`Attempt to finish off a wounded foe, causing ${$m1+$AP*0.2} damage and converting each extra point of rage into $*10;F1 additional damage (up to a maximum cost of 30 rage).  Only usable on enemies that have less than 20% health.`

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1617549311457174700.sql (hunk 1 above)
Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1617549311457174700.sql:19]: ok :+1:

Comment (Stifler82) [reply data/sql/updates/pending_db_world/rev_1617549311457174700.sql:19]: > I am not sure that spell_dbc description fields affect the description that you see in the client, but that's ok

Yeah i confirm, it doesn't affect the client description, but if a day we will find out a way to do it, we will have a lot of bullshit wrote there, an then we will have fix the texts as well.. i would like avoid it..

**Comment by Si1ker** [comment]
same as https://github.com/azerothcore/azerothcore-wotlk/pull/5171#issuecomment-814640177

**Comment by Algorak** [comment]
Hello I have a question this error also happens when doing magic like spell = 47847 shadow fury from the witcher. Does that fix the bug or does this fix only affect warriors?


It happens quite often that the spell = 47847 shadow fury just has no effect. It doesn't fail or resist. There is simply no information in the log ?! This seems to affect some classes, however, where spells simply disappear in the matrix and have no effect.

**Pull request merged**